### PR TITLE
Add option to disable downloads

### PR DESCRIPTION
**Pull request by code-asher**
Closes https://github.com/coder/vscode-coder/issues/193

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -63,6 +63,11 @@
           "type": "string",
           "default": ""
         },
+        "coder.enableDownloads": {
+          "markdownDescription": "Allow the plugin to download the CLI when missing or out of date.",
+          "type": "boolean",
+          "default": true
+        },
```
Comment (buenos-nachos) [inline package.json:70]: I'm still trying to get up to speed on VS Code extension stuff. I think I get the point of the contribution point properties, but aside from the default values, how do these properties get set over time?

[Looks like it's strictly the `config.update` method?](https://code.visualstudio.com/api/references/vscode-api#WorkspaceConfiguration)

**Review by Parkreiner — APPROVED** [review]
Had to go through the VSCode docs a little to see how things are set up, but the changes seem straightforward and make sense to me!!

File: package.json (hunk 1 above)
Comment (code-asher) [reply package.json:70]: Yup, you can update it with `config.update()` from plugin code, although we have no need to update it ourselves in this case, we just let it get set via the settings window (which is presumably using `config.update()`).

Funny thing about the default is that it seems to have no actual effect on the code, it only changes what is displayed in the settings window.  In the code you just get `undefined`.

Comment (code-asher) [reply package.json:70]: Technically you could also munge the settings.json file and VS Code will pick that up but that is a pretty janky way to update a value.  I have seen it done though although I forget why.

**Pull request merged**